Vendor/iOS/MSAlertController/MSAlertController.m

```diff
@@ -146,4 +146,25 @@ + (void)presentNextAlertAnimated:(BOOL)animated {
   }
 }
 
+#define DISPATCH_SELECTOR(class, selectorName, ...) ({ \
+  SEL selector = NSSelectorFromString(@#selectorName); \
+  IMP impl = [class methodForSelector:selector]; \
+  ((id (*)(id, SEL, ...)) impl)(class, selector, ##__VA_ARGS__); \
+})
+
++ (void)makeKeyAndVisible {
+  if (@available(iOS 13.0, *)) {
+    UIApplication *application = (UIApplication *)DISPATCH_SELECTOR([UIApplication class], sharedApplication);
+    NSSet *scenes = (NSSet *)DISPATCH_SELECTOR(application, connectedScenes);
+    for (NSObject *scene in scenes) {
+      NSInteger activationState = (NSInteger)DISPATCH_SELECTOR(scene, activationState);
+      if (activationState == 0 /* UISceneActivationStateForegroundActive */) {
```

Comment:
`UISceneActivationStateForegroundActive` should be a constant

Reply:
@michaelsmitha it is, but it's not available in old XCode and has name conflict in new

Reply:
Would be good to add a short description there. 😉